doc/reference/pipeline_spec.md

```diff
@@ -31,7 +31,11 @@ create-pipeline](../pachctl/pachctl_create-pipeline.html) doc.
     "constant": int
     "coefficient": double
   },
-  "resource_spec": {
+  "resource_requests": {
+    "memory": string
```

Comment:
I think there should be a comma here, although it looks like it was already missing.

Reply:
Ok ... I was trying to figure out what was wrong ... didn't realize GH was smart enough to do syntax linting on json code snippets ... thats nifty